Source/DafnyCore/Resolver/NameResolutionAndTypeInference/NameResolutionAndTypeInference.cs

```diff
@@ -3327,7 +3327,7 @@ internal void ResolveActualParameters(ActualBindings bindings, List<Formal> form
             whatKind + (context is Method ? " in-parameter" : " parameter"));
 
           AddAssignableConstraint(
-            callTok, formal.Type.Subst(typeMap), b.Actual.Type,
+            callTok /* TODO should be b.Actual.Origin */, formal.Type.Subst(typeMap), b.Actual.Type,
```

Comment:
TODO for this PR or for a subsequent PR? Or a different issue?

Reply:
When using `callTok`, if there is a problem with one of the arguments, then the error is reported on the call `(` instead of on the center of the argument. The TODO is meant for a follow-up.